convert2rhel/checks.py

```diff
@@ -293,7 +294,15 @@ def get_unsupported_kmods(host_kmods, rhel_supported_kmods):
     Ignore certain kmods mentioned in the system configs. These kernel modules moved to kernel core, meaning that the
     functionality is retained and we would be incorrectly saying that the modules are not supported in RHEL."""
     return (
-        host_kmods
-        - rhel_supported_kmods
-        - set(system_info.kmods_to_ignore)
+            host_kmods
+            - rhel_supported_kmods
+            - set(system_info.kmods_to_ignore)
     )
+
+
+def check_repos_are_valid():
```

Comment:
This function runs under all conditions. That is even when `--disable-submgr` is not used. Didn't you want to check only custom repos?

Reply:
Not resolved yet.
Please add a condition right after the line `logger.task("Prepare: Checking if --enablerepo repositories are accessible")` that would print an `info` message and return from the function.

Something like:
```
if not tool_opts.disable_submgr:
    logger.info("Skipping the check of repositories due to the use of RHSM for the conversion.")
    return
```